### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Alexandre Iorio

### DISCUSSION THREAD

**Discussion by AlexandreIorio** [heig-vd-dai-course/heig-vd-dai-course#127] [Show and tell]
Here is my GitHub repository: [https://github.com/AlexandreIorio/DAI_Docker/tree/main#what-is-the-difference-between-a-container-and-an-image](https://github.com/AlexandreIorio/DAI_Docker/tree/main#what-is-the-difference-between-a-container-and-an-image)

![package](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/90898699/36b72b9c-88db-40bf-ab1f-0a836d6c69e3)

**Comment by ludelafo** [comment]
Everything looks good to me!
